libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -454,7 +454,6 @@ ASTPointer<VariableDeclaration> ASTJsonImporter::createVariableDeclaration(Json:
 		nullOrCast<Expression>(member(_node, "value")),
 		visibility(_node),
 		_node["documentation"].isNull() ? nullptr : createDocumentation(member(_node, "documentation")),
-		memberAsBool(_node, "stateVariable"),
```

Comment:
We are still exporting it. Why don't we remove this member?

Reply:
I'll create a PR against breaking to remove it.